lib/internal/vm.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const callbackMap = new WeakMap();
+
+module.exports = { callbackMap };
```

Comment:
This should be in a separate file under `lib/internal/vm/` like everything else.